OxfordBodleian/Juel-Jensen/BDLaethf28.xml

```diff
@@ -96,9 +101,16 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                             </item>
                                         </list>
                                 </collation>
-                                
+                                <condition key="good">
+                                 The bookblock has been damaged by humidity, especially in the outer margins.
+                                 Hairs are still attached to <locus target="#49"/>.
+                                 Insect droppings are present on <locus target="#31 #32"/>. The blunt edge used for ruling has perforated <locus target="#22"/>.
```

Comment:
I have a question regarding the use of f. Does the f. will be prompt in the PDF automatically or should we add it? We were discussing about that also with @MassimoVilla.

Reply:
It will be prompted automatically. This is complex, so there will be many errors that we'll have to fix manually in the end. But I think it's better to not insert it into the files now to increase consistency, which will make find/replace later easier.

Reply:
:(